packages/preload/src/index.ts

```diff
@@ -2260,6 +2260,14 @@ export function initExposure(): void {
       return ipcInvoke('docker-compatibility:getSystemDockerSocketMappingStatus');
     },
   );
+
+  contextBridge.exposeInMainWorld('getDockerContexts', async (): Promise<DockerContextInfo[]> => {
+    return ipcInvoke('docker-compatibility:listDockerContexts');
+  });
+
+  contextBridge.exposeInMainWorld('switchDockerContext', async (contextName: string): Promise<DockerContextInfo[]> => {
+    return ipcInvoke('docker-compatibility:switchDockerContext', contextName);
```

Comment:
Minor: I think I'd prefer setDockerContext. I know there are typically going to be 2, but switch sounds like toggling between two states, where set sounds like picking from the list.

Reply:
there is always the default state, there is never an unknown state